orchestrator/orchestrator.js

```diff
@@ -0,0 +1,62 @@
+const request = require('request-promise')
```

Comment:
There will be a few linter errors when you place this file inside of the `server/orchestrator` folder. Run `npm run lint` to find them.